src/lib.rs

```diff
@@ -206,7 +206,9 @@ if_std! {
 
     pub mod task;
     pub mod executor;
+    pub mod local_executor;
```

Comment:
Oh sorry I more meant that this could be used in tests, we should avoid exposing it in the crate for now